plugins/org.jboss.tools.openshift.common.ui/src/org/jboss/tools/openshift/internal/common/ui/application/importoperation/MavenProjectImportOperation.java

```diff
@@ -53,9 +59,38 @@ public void setFilters(Collection<String> filters) {
 		MavenModelManager modelManager = mavenPlugin.getMavenModelManager();
 		Set<MavenProjectInfo> projectInfos = getMavenProjects(getProjectDirectory(), filters, modelManager, monitor);
 		ProjectImportConfiguration projectImportConfiguration = new ProjectImportConfiguration();
-		List<IMavenProjectImportResult> importResults =
-				configurationManager.importProjects(projectInfos, projectImportConfiguration, monitor);
-		return validate(toProjects(importResults));
+		if (overwriteMavenProjects(projectInfos, projectImportConfiguration, monitor)) {
+	        List<IMavenProjectImportResult> importResults =
+	                configurationManager.importProjects(projectInfos, projectImportConfiguration, monitor);
+	        return validate(toProjects(importResults));
+		} else {
+		    return Collections.EMPTY_LIST;
```

Comment:
replace by ```Collections.emptyList()``` to avoid a compiler warning about unchecked conversion to List<IProject>